ui/v2.5/src/docs/en/Tasks.md

```diff
@@ -8,7 +8,7 @@ The scan function walks through the stash directories you have configured for ne
 
 Stash currently identifies files by performing a quick file hash. This means that if the file is renamed for moved elsewhere within your configured stash directories, then the scan will detect this and update its database accordingly.
 
-Stash currently ignores duplicate files. If a file is detected with the same hash as a file already in the database (and that file still exists on the filesystem), only the last copy is used.
+Stash currently ignores duplicate files. If two files contain identical content, only the last one it comes across is used.
```

Comment:
`If two files contain identical content, only the last one it comes across is used.`
Isnt it the opposite? In case of multiple files with the same hash the first file will be added during scan (if its not in the db) and all the rest will be considered as duplicates and be ignored.

Reply:
Had to consult the code, but it seems you are correct. Fixed.